styles.css

```diff
@@ -19,49 +31,403 @@
     box-sizing: border-box;
 }
 /* ------------------------------- 1. Global Styles -------------------------------- */
+body {
+    background-color: white;
+}
+header {
+    color: var(--licht-grijs);
+    background: linear-gradient(-45deg, var(--paars), var(--blauw)), url(./assets/noise-texture.png);
+    background-blend-mode: color;
+
+}
+main {
+    color: var(--zwart);
+}
+footer {
+    background-color: var(--zwart);
+    color: var(--licht-grijs);
+}
+
+button {
+    background-color: var(--roze);
+    color:  var(--licht-grijs);
+    border: none;
+    height: 50px;
+    width: fit-content;
+    padding: 0 60px 0 60px;
+    border-radius: 25px;
+    box-shadow: 0px 3px 3px  rgba(0, 0, 0, 0.50);
+    text-transform: uppercase;
+}
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1, h2, h3, h4,h5,h6 {
+    font-family: Merriweather, serif;
+}
+
+h1 {
+    font-size: 30px;
+}
+
+p, a, a:link, a:visited, li, label, textarea {
+    font-family: Roboto, sans-serif;
+    text-decoration: none;
+    font-size: 15px;
+    /*z-index: 0;*/
+}
+
+ul {
+    list-style: none
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+/* --------------- 3.1 Reusables ----------------------- */
+:root {
+    --paars: rgba(87,34,205,1);
+    --blauw:  rgba(75,73,232,1);
+    --roze: #ED4385;
+    --zwart:  #0C0C0C ;
+    --licht-paars: #F2F0FF;
+    --licht-grijs: #F3F6F9;
+    --grijs:  #7A7A7A;
+    --page-width: 1200px;
```

Comment:
Is er een speciale reden waarom je hebt gekozen om een gefixeerde breedte te pakken?